sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/command/DynamicAgentArgsManager.java

```diff
@@ -52,4 +56,22 @@ public static void refreshAgentArgs(Map<String, String> newAgentArgs) {
     public static String getAgentArg(String key) {
         return AGENT_ARGS.get(key);
     }
+
+    /**
+     * get dynamic plugin path
+     *
+     * @return dynamic plugin path
+     */
+    public static String getDynamicPluginPackagePath() {
+        return AGENT_ARGS.get(SERMANT_PLUGIN_PACKAGE_PATH_KEY);
```

Comment:
decide to use (CommonConstant.AGENT_FILE_KEY)

Reply:
done